contrib/resources/translations/README.md

```diff
@@ -0,0 +1,98 @@
+
+# General translation workflow
+
+## Preparing file with English strings
+
+First, you need an up to date file with English strings. The one present in this
+directory is most likely outdated; to generate the current one either execute:
+
+```
+dosbox -lang en -c "config -wl en.lng" -c "exit"
+```
+
+or do the following:
+
+- In the configuration file, go to the `[dosbox]` section and set`language = en`.
+- Run the most current version of DOSBox Staging.
+- Inside DOSBox execute the command `config -wl en.lng` (you can supply a host OS
+  path, instead of simply `en.lng`).
+
+This will produce a file with the current strings to translate, use it as a
+reference.
+
+## Updating the translation
+
+To update translation just edit the relevant `*.lng` file with a text editor and save
+the file as UTF-8 encoded with Unix line endings.
```

Comment:
Small suggestion:

Update a given translation by editing the corresponding `*.lng` file with a text editor and saving it with UTF-8 encoding and Unix line endings.

Reply:
OK, changed.